app.js

```diff
@@ -121,4 +93,55 @@ function renderAll() {
   }
 };
 
+//Event Handler
+function addNewStore(event) {
+  event.preventDefault();
+
+  if (!event.target.name.value || !event.target.minCustomers.value || !event.target.maxCustomers.value || !event.target.avgSale.value) {
+    return alert('Oops, you forgot something! Please check your entries and try again.');
+  }
+
+  if(addMinCustomers < 0 || addMaxCustomers < 0 || addAvgSale) {
```

Comment:
None of these if statements trigger, as you are referencing conditions where the comparison doesnt exist